### ISSUE TITLE
Weybourne in importTraj

### ISSUE THREAD

**Issue by lcrilleyl** [davidcarslaw/openair#51]
Hi,
Could i please get Weybourne Atmospheric Observatory added to the sites for importTraj.
Thanks
Leigh

**Comment by davidcarslaw** [comment]
yes, can do. Which year(s) are you interested in? I can run this up to and including 2015 with the reanalysis met data.

**Comment by lcrilleyl** [comment]
Great thanks. I am interested in 2015.

**Comment by davidcarslaw** [comment]
Should now be available on the server. Note, you need a recent version of openair, as the server address has changed in the code:

```traj <- importTraj("weybourne", 2015)```

**Comment by lcrilleyl** [comment]
Hi David,
Thanks a lot for adding Weybourne.
I have used the code below to import the traj data but I have noticed that for some key dates that I am interested in, the back trajectories don’t seem to have loaded properly. Below is an example, and it appears to be for 29th june till 4th July 2015  that it does not load properly.

Any help much appreciated.

> trajPlot(selectByDate(traj, start = "29/6/2015", end ="30/6/2015"))
Error in seq.default(x1, x1, length.out = n) :
  'from' cannot be NA, NaN or infinite
In addition: Warning messages:
1: In min(x) : no non-missing arguments to min; returning Inf
2: In max(x) : no non-missing arguments to max; returning -Inf
3: In min(x) : no non-missing arguments to min; returning Inf
4: In max(x) : no non-missing arguments to max; returning -Inf
5: In min(x) : no non-missing arguments to min; returning Inf
6: In max(x) : no non-missing arguments to max; returning -Inf
